helm/cert-exporter-chart/templates/daemonset.yaml

```diff
@@ -40,23 +38,21 @@ spec:
             cpu: 50m
             memory: 50Mi
         volumeMounts:
-        - mountPath: {{ .Values.Installation.V1.Monitoring.CertExporter.CertPath }}
+        - mountPath: /etc/kubernetes/ssl
```

Comment:
We don't currently support the templating we're using here when deploying to tenant clusters, and we don't support the templating we're using when deploying to tenant clusters when deploying to host clusters.

Thankfully, these values should be the same between control planes and tenant clusters now.